rules/rest-api-design/rule.md

```diff
@@ -19,7 +19,7 @@ redirects: []
 
```

Comment:
Hey @sethdaily 

- [ ] 1. Please add the old URL under "redirects:" so link won't be broken

Reply:
@sethdaily 

I still don't see the old URL in redirects...

Reply:
it's there
- key-principles-of-rest-api-design